src/main/java/com/google/devtools/build/lib/rules/cpp/Link.java

```diff
@@ -156,6 +156,14 @@ private Link() {} // uninstantiable
         ArtifactCategory.INTERFACE_LIBRARY,
         Executable.NOT_EXECUTABLE),
 
+    /** A pdb file for dynamic library. */
+    PDB_FILE(
+        LinkerOrArchiver.LINKER,
+        CppActionNames.CPP_LINK_DYNAMIC_LIBRARY,
```

Comment:
Don't know how to declare output for both CPP_LINK_EXECUTABLE and CPP_LINK_DYNAMIC_LIBRARY in one enum. But seems it works fine.

Can someone take an advice about right way to fix that?

Reply:
Actully not used now.